pkg/scheduler/api/job_info.go

```diff
@@ -704,6 +704,9 @@ func (ji *JobInfo) CheckTaskMinAvailable() bool {
 
 	klog.V(4).Infof("job %s/%s actual: %+v, ji.TaskMinAvailable: %+v", ji.Name, ji.Namespace, actual, ji.TaskMinAvailable)
 	for task, minAvailable := range ji.TaskMinAvailable {
+		if minAvailable <= 0 {
```

Comment:
Why not intercept invalid values ​​via webhook?

Reply:
Why can it be an invalid value? I agree with @hwdef that it should be checked in the webhook.

Reply:
when task replica == 0，then minAvailable will be zero.

and is valid to set task replica to 0

Reply:
/cc @william-wang How do you think about it? Is there some scenarios that users may set `task.replicas` to be `0`?

Reply:
task.Replicas is allowed to be `0`.  Please do this kind of check in webhook.

Reply:
done